string interacter_frame
uint32 object_id
uint8 action
uint8 PICK_UP = 0
uint8 DROP = 1
---
bool success
string error_message
